client/lost-app/src/styles/App.css

```diff
@@ -6,7 +6,29 @@
   height: 40vmin;
   pointer-events: none;
 }
-
+.recentPlayed{
+  box-shadow: 0 0 50px #ccc;
+  width: 300px;
+  height: 300px;
+}
+.recentAttributes{
+  box-shadow: 0 0 50px #ccc;
+  width: 300px;
+  height: 300px;
+ 
```

Comment:
potentially just give recentPlayed and recentAttributes the same class name because the css is the same